leakcanary-android-core/src/main/java/leakcanary/internal/LeakCanaryFileProvider.kt

```diff
@@ -31,28 +31,26 @@ import android.os.StrictMode
 import android.provider.OpenableColumns
 import android.text.TextUtils
 import android.webkit.MimeTypeMap
-import org.xmlpull.v1.XmlPullParser.END_DOCUMENT
-import org.xmlpull.v1.XmlPullParser.START_TAG
-import org.xmlpull.v1.XmlPullParserException
 import java.io.File
 import java.io.FileNotFoundException
 import java.io.IOException
 import java.util.HashMap
+import org.xmlpull.v1.XmlPullParser.END_DOCUMENT
+import org.xmlpull.v1.XmlPullParser.START_TAG
+import org.xmlpull.v1.XmlPullParserException
 
 /**
  * Copy of androidx.core.content.FileProvider, converted to Kotlin.
  */
 internal class LeakCanaryFileProvider : ContentProvider() {
 
-  private var mStrategy: PathStrategy? = null
+  private lateinit var mStrategy: PathStrategy
```

Comment:
Make `mStrategy` lateinit instead of nullable, as it was called with assert for times.